addons/l10n_it/data/account_chart_template.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="l10n_it_chart_template_generic" model="account.chart.template">
+        <field name="name">IT Tax and Account Chart Template</field>
+        <field name="bank_account_code_prefix">1200</field>
+        <field name="cash_account_code_prefix">1210</field>
+        <field name="transfer_account_code_prefix">1220</field>
+        <field name="code_digits">6</field>
+        <field name="currency_id" ref="base.EUR"/>
+        <field name="country_id" ref="base.ie"/>
+    </record>
```

Comment:
why changing that? Original values:
```
        <record id="l10n_it_chart_template_generic" model="account.chart.template">
            <field name="name">Italy - Generic Chart of Accounts</field>
            <field name="cash_account_code_prefix">180</field>
            <field name="bank_account_code_prefix">182</field>
            <field name="transfer_account_code_prefix">183</field>
            <field name="currency_id" ref="base.EUR"/>
            <field name="country_id" ref="base.it"/>
        </record>
```
Note: big typo in `<field name="country_id" ref="base.ie"/>`

Reply:
Dang! Fixing it NOW.

Reply:
:fire: :fire: :ambulance: :hospital: :fire: :fire: